recipes/tests.rb

```diff
@@ -218,8 +218,30 @@ module load intelmpi && mpirun --help | grep '#{node['cluster']['intelmpi']['kit
 end
 
 ###################
-# EFA - GDR (GPUDirect RDMA)
+# EFA
 ###################
+if node['cluster']['os'].end_with?("-custom")
```

Comment:
please check with @lukeseawalker if we want to keep this test here or move it to the image builder validation phase

Reply:
I think it does add value if check is moved to build validation process

Reply:
Removed from cookbook, it will be part of another patch for the CLI.